src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -42,13 +42,15 @@ import {
 } from "./format";
 import { PreTableControls, parcelTableColumnStyleOptions } from "./styles";
 import { DbParcelRow } from "@/databaseUtils";
+import { searchForBreakPoints } from "./conditionalStyling";
 
 const ParcelsPage: React.FC<{}> = () => {
     const [isLoading, setIsLoading] = useState(true);
     const [parcelsDataPortion, setParcelsDataPortion] = useState<ParcelsTableRow[]>([]);
     const [filteredParcelCount, setFilteredParcelCount] = useState<number>(0);
     const [selectedParcelId, setSelectedParcelId] = useState<string | null>(null);
     const [clientIdForSelectedParcel, setClientIdForSelectedParcel] = useState<string | null>(null);
+    const [parcelRowBreakPoints, setParcelRowBreakPoints] = useState<number[][]>([[], []]);
```

Comment:
I think the [][] type might not be the clearest here. Eg., by looking at this file, I don't really know whether the outer array corresponds to the packing slots, or the inner array. Also, this isn't very sustainable if we wanted to add more dividing lines. Can you think of a better data structure to use here? we want it to 1. be clear which array of numbers corresponds to which column that we're using to find the breakpoints and 2. be sustainable if we wanted to add more dividing lines and 3. cope with the case where there are dividing lines between the same 2 rows according to more than one measure (ie between 2 dates the packing slot also changes, we need to cope with that)